ecs-service/security-group.tf

```diff
@@ -0,0 +1,35 @@
+resource "aws_security_group" "this" {
+  name        = "${var.environment}-${var.service_name}-ecs"
+  description = "${var.environment}-${var.service_name} ECS container security group"
+  vpc_id      = var.vpc_id
+
+  tags = {
+    Name = "${var.environment}-${var.service_name}-ecs"
+  }
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_security_group_rule" "allow_outbound" {
+  description       = "All all outbound"
+  security_group_id = aws_security_group.this.id
+
+  type        = "egress"
+  from_port   = 0
+  to_port     = 0
+  protocol    = "-1"
+  cidr_blocks = ["0.0.0.0/0"]
```

Comment:
`cidr_blocks = ["0.0.0.0/0"]` ?

Reply:
Yes, allow connecting to everywhere. 
At the moment I don't see how we would track IP addresses or ranges for each service and each of the 3rd party service it uses. Ie connecting to github api, twitter api, fb api etc... 

I understand that limiting per range or /32 is safer, it's just a lot of work right now. 

But if you have a different view we make this a mandatory parameter.

Reply:
No, I think it's totally OK to allow using Internet services by default.
I think I should have asked this: Isn't this supposed to be going through the NAT gw and therefore only needs direct access to the gw? or not because there is something else I'm not considering, like aws internal services?

Reply:
I see! Yes we could think about it that way too. 
Do you want me to add a variable for natgws/igws?

Reply:
yes, I think some limit would be healthy